build/openwisp_freeradius/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM freeradius/freeradius-server:3.0.19-alpine
+
+RUN apk add --no-cache --update postgresql-dev postgresql-client && \
+    rm -rf /var/cache/apk/* /tmp/*
+
+ENTRYPOINT ["sh", "init_command.sh"]
+EXPOSE 1812/udp 1813/udp
+
+COPY ./common/init_command.sh \
+    ./common/utils.sh ./
+COPY --chown=root:root ./openwisp_freeradius/raddb/ /etc/raddb/
+
+ENV MODULE_NAME=freeradius \
+    DASHBOARD_DOMAIN=dashboard.example.com \
+    DB_HOST=postgres \
+    DB_PORT=5432 \
+    DB_USER=admin \
+    DB_PASSWORD=admin \
+    DB_DATABASE=openwisp_db \
+    DB_SSLMODE=disable \
```

Comment:
Did we decide to drop database SSL as an option?  I know we talked about it.  If so then `DB_SSLMODE` option can be removed.

Reply:
```
    export DB_SSLMODE=`echo "$DB_OPTIONS" | jq -r '.sslmode'`
    if [ "$DB_SSLMODE" == "null" ]; then
        export DB_SSLMODE=disable
    fi
```

The `DB_OPTIONS` is a json variable taking care of additional database variables, I used that to take care about the SSL settings with database.
(Are you reffering to the discussion here: https://groups.google.com/d/msg/openwisp/Mh0tMBrB2Js/0cXVIr62AQAJ or some other discussion?)

Reply:
Thanks, I missed that the variables had moved to DB_OPTIONS.